src/js/tech/html5.js

```diff
@@ -70,6 +72,13 @@ class Html5 extends Tech {
       }
     }
 
+    if (this.featuresNativeVideoTracks) {
+      this.handleVideoTrackChange_ = Fn.bind(this, this.handleVideoTrackChange);
```

Comment:
This is inline with how the native text tracks are done. The reason for them is that in one vendor, the functions weren't being properly bound if we were using `this.handle<Thing>Track<Action>` directly. Binding them here and then using the bound version in the listener fixes it.